src/api/app/models/event/request.rb

```diff
@@ -81,37 +81,30 @@ def payload_with_diff
 
       ret = payload
       payload['actions'].each do |a|
-        diff = calculate_diff(a)
+        diff = calculate_diff(a).try(:lines)
         next unless diff
-        diff = diff.lines
-        dl = diff.length
-        if dl > DiffLimit
-          diff = diff[0..DiffLimit]
-          diff << "[cut #{dl - DiffLimit} lines to limit mail size]"
+        diff_length = diff.length
+        if diff_length > DIFF_LIMIT
+          diff = diff[0..DIFF_LIMIT]
+          diff << "[cut #{diff_length - DIFF_LIMIT} lines to limit mail size]"
         end
         a['diff'] = diff.join
       end
       ret
     end
 
     def reviewers
-      ret = []
-      BsRequest.find_by_number(payload['number']).reviews.each do |r|
-        ret.concat(r.users_and_groups_for_review)
-      end
-      ret.uniq
+      BsRequest.find_by_number(payload['number']).reviews.map(&:users_and_groups_for_review).flatten.uniq
     end
 
     def creators
       [User.find_by_login(payload['author'])]
     end
 
     def action_maintainers(prjname, pkgname)
-      ret = []
-      payload['actions'].each do |a|
-        ret.concat _roles('maintainer', a[prjname], a[pkgname])
-      end
-      ret.uniq
+      payload['actions'].map do |action|
+        _roles('maintainer', action[prjname], action[pkgname])
+      end.flatten.uniq
```

Comment:
Because it is really ugly :see_no_evil: It is much more difficult to read, and new lines are for free :wink: 

According to [Ruby Style guide](https://github.com/bbatsov/ruby-style-guide#single-line-blocks): Avoid `do...end` when chaining

Reply:
> Ana06 18 hours ago
> Because it is really ugly see_no_evil It is much more difficult to read, and new lines are for free wink

> According to Ruby Style guide: Avoid do...end when chaining

That's what I tried, because I also don't like this to much. But then Rubocop complains :wink:

So let's stick to our style guide :wink: